docs/reference/command-notebook-config.txt

```diff
@@ -42,6 +42,11 @@ The following configuration settings are supported:
   command/notebook. This does not need to be unique. The default
   description consists of a timestamp and the entrypoint of the
   command.
+
+- ``entrypoint``: Specifies a list of optional arguments for the container
```

Comment:
blocking: i think it would be more accurate to say `list of optional arguments for launching tensorboard` as the fact that the arguments are passed to the container is an implementation detail of our system.

non-blocking: renaming this to `tensorboard_args` seems more appropriate